server/src/lib/model/db/import-locales.ts

```diff
@@ -112,6 +112,56 @@ const VARIANTS: Variant[] = [
     variant_name: 'Portuguese (Portugal)',
     variant_token: 'pt-PT',
   },
+  {
+    locale_name: 'ca',
+    variant_name: 'Central',
+    variant_token: 'ca-central',
+  },
+  {
+    locale_name: 'ca',
+    variant_name: 'Valencià',
+    variant_token: 'ca-valencia',
+  },
+  {
+    locale_name: 'ca',
+    variant_name: 'Balear',
+    variant_token: 'ca-balear',
+  },
+  {
+    locale_name: 'ca',
+    variant_name: 'Nord-Occidental',
+    variant_token: 'ca-nwestern',
+  },
+  {
+    locale_name: 'ca',
+    variant_name: 'Septentrional',
+    variant_token: 'ca-northern',
+  },
+  {
+    locale_name: 'ca',
+    variant_name: 'Tortosí',
+    variant_token: 'ca-valencia-tortosi',
+  },
+  {
+    locale_name: 'ca',
+    variant_name: 'Valencià central',
+    variant_token: 'ca-valencia-central',
+  },
+  {
+    locale_name: 'ca',
+    variant_name: 'Valencià septentrional',
+    variant_token: 'ca-valencia-northern',
+  },
+  {
+    locale_name: 'ca',
+    variant_name: 'Valencià meridional',
+    variant_token: 'ca-valencia-southern',
+  },
+  {
+    locale_name: 'ca',
+    variant_name: 'Alacantí',
+    variant_token: 'ca-valencia-alacant',
+  },
```

Comment:
Could we add one more:

```
 {
    locale_name: 'ca',
    variant_name: 'Alguerès',
    variant_token: 'ca-algueres',
  },
```
This is for the variant spoken on the isle of Sardinia. There is a campaign planned there in the near future.